THEOplayer-Connector-Comscore.podspec

```diff
@@ -16,9 +16,11 @@ Pod::Spec.new do |s|
   s.platforms    = { :ios => "13.0", :tvos => "13.0" }
 
   s.source_files = 'Code/Comscore/Source/**/*'
+  s.resource_bundles = {'THEOplayer-Connector-Comscore' => ['version.json']}
       
   s.static_framework = true
   s.swift_versions = ['5.3', '5.4', '5.5', '5.6', '5.7']
-  s.dependency 'THEOplayerSDK-core', "~> 8"
   s.dependency 'ComScore', '~> 6.10.0'
+  s.dependency 'THEOplayerSDK-core', "~> 8"
+  s.dependency 'THEOplayer-Connector-Utilities', "~> " + theoplayer_connector_major_minor_version, ">= " + theoplayer_connector_version
```

Comment:
there are some Utilities-specific checks in our GItHub action pipeline.
Were those already taken into account?